tmt/steps/discover/fmf.py

```diff
@@ -250,17 +306,6 @@ class DiscoverFmf(tmt.steps.discover.DiscoverPlugin[DiscoverFmfStepData]):
             how: fmf
             dist-git-source: true
 
-    Related config options (all optional):
```

Comment:
At the beginning of of the `fmf` plugin section we still have [the warning](https://tmt--3311.org.readthedocs.build/en/3311/plugins/discover.html#fmf):

    Please, be aware that the documentation below is a work in progress...

Shall we remove it now that all missing bit should be there? @happz or do we still need to fill more gaps here?

Reply:
I would leave it as it is, this patch is moving things from one place to another. I'm afraid most of our plugins still need a review of their own docs, marking literals as literals, adding links, make metavars better, and so on.

Reply:
Ack.